src/gwt/acesupport/acemode/markdown_highlight_rules.js

```diff
@@ -65,7 +65,7 @@ var escaped = function(ch) {
 function github_embed(tag, prefix) {
     return { // Github style block
         token : "support.function",
-        regex : "^\\s*```" + tag + "\\s*$",
+        regex : "^\\s*```\\{?" + tag + "([^\\}]*\\})?\\s*$",
```

Comment:
I think you need to ensure that only non-capturing regex groups are used here.

FWIW I think this also implies that the IDE would accept e.g.

    ```{js
    ```js}

when highlighting a block -- is that okay?